packages/workflows/src/definition/carts/add-shipping-method/index.ts

```diff
@@ -0,0 +1,69 @@
+import { createWorkflow, transform } from "../../../utils/composer"
```

Comment:
praise(this-file): Mad props @carlos-r-l-rodrigues and @adrien2p. This was a pleasure to work with